src/renderer_gl.cpp

```diff
@@ -2566,6 +2572,13 @@ namespace bgfx { namespace gl
 						setTextureFormat(TextureFormat::D0S8,  GL_STENCIL_INDEX8, GL_STENCIL_INDEX, GL_UNSIGNED_BYTE); // Only works as renderbuffer, not as texture
 					}
 				}
+				else
+				{
+#ifdef GL_BGR
```

Comment:
No ifdefs please. Define `GL_BGR` in renderer_gl.h the same way `GL_BGRA` is defined:
https://github.com/bkaradzic/bgfx/blob/30f2920ae5b27ca6c00c5263515b6fff9dbe723f/src/renderer_gl.h#L165-L167

Reply:
Oh yeah, that's much cleaner